.circleci/config.yml

```diff
@@ -12,6 +12,18 @@ jobs:
       - run:
           name: validate
           command: npm run check-examples
+  test_examples_2:
```

Comment:
Maybe rename to test_examples_python and test_examples_node (above) so that it is clearer from the PR directly which tool failed?